Code/Framework/AzCore/AzCore/std/string/conversions.h

```diff
@@ -286,6 +286,13 @@ namespace AZStd
         str = buf;
     }
 
+    template<class Str, class BoolType>
+    auto to_string(Str& str, BoolType value)
+        -> enable_if_t<same_as<remove_cvref_t<BoolType>, bool>>
+    {
+        str = value ? "true" : "false";
```

Comment:
It could be nice to look through the existing code for places that do `"true" : "false"` and update them to use this function. But we can do that later.